### ISSUE TITLE
serverless-stack depends on a vulnerable version of netmask via CDK (CVE-2021-28918)

### ISSUE THREAD

**Issue by krivachy** [serverless-stack/serverless-stack#282]
CDK already has an open issue to update their dependency: https://github.com/aws/aws-cdk/issues/13873

After CDK releases a new version it would be good if serverless-stack updated to the new non-vulnerable version.

CVE details: https://github.com/advisories/GHSA-pch5-whg9-qr2r

```
➜  npm ls netmask
test@0.0.0
└─┬ @serverless-stack/cli@0.10.6
  └─┬ aws-cdk@1.94.1
    └─┬ proxy-agent@4.0.1
      └─┬ pac-proxy-agent@4.1.0
        └─┬ pac-resolver@4.1.0
          └── netmask@1.0.6 
```

**Comment by fwang** [comment]
Thanks @krivachy. Will do. I will keep an eye on this.

**Comment by fwang** [comment]
Got it! I will up the version in the next release, and resolve the breaking changes.

**Comment by fwang** [comment]
Released v0.10.11 with CDK 1.98.0
